### PR TITLE
Handle and ignore an obscure corner case in the frame projection unit tests.

### PR DESCRIPTION
**Pull request by JonathanLennox**

### CODE REVIEW

File: src/test/java/org/jitsi/videobridge/cc/vp8/VP8AdaptiveTrackProjectionTest.java [hunk 1]
```diff
@@ -124,121 +124,137 @@ private void runInOrderTest(Vp8PacketGenerator generator, int targetIndex)
...
+        for (int i = 0; i < reorderSize; i++)
+        {
+            buffer.add(generator.nextPacket());
+        }
 
-            for (int i = 0; i < reorderSize; i++)
-            {
-                buffer.add(generator.nextPacket());
-            }
+        Random random = new Random(seed);
```
Comment (bbaldino) [inline src/test/java/org/jitsi/videobridge/cc/vp8/VP8AdaptiveTrackProjectionTest.java]: kotlintest actually has some support for this (random but including some pre-defined constants), maybe that'd work well here? https://github.com/kotlintest/kotlintest/blob/master/doc/reference.md#custom-generators

Comment (JonathanLennox) [reply src/test/java/org/jitsi/videobridge/cc/vp8/VP8AdaptiveTrackProjectionTest.java]: Well, this is a Java test case...

Comment (bbaldino) [reply src/test/java/org/jitsi/videobridge/cc/vp8/VP8AdaptiveTrackProjectionTest.java]: Oh, oops.

**Comment by JonathanLennox** [comment]
BTW because indentation changed this looks like a lot more changed than really did; I suggest reviewing the diff with the "hide whitespace changes" checkbox set.
